src/components/PersonInfo.tsx

```diff
@@ -0,0 +1,70 @@
+import { useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../types';
+import { PersonLink } from './PersonLink';
+
+interface Props {
+  key: string;
+  person: Person;
+  mother: Person | null;
+  father: Person | null;
+}
+
+export const PersonInfo: React.FC <Props> = ({
+  key,
+  person,
+  mother,
+  father,
+}) => {
+  const {
+    name,
+    sex,
+    born,
+    died,
+    fatherName = '-',
+    motherName = '-',
+    slug,
+  } = person;
+
+  const { slectedPersonSlug = '' } = useParams();
+
+  const isSelected = (personSlug: string) => personSlug === slectedPersonSlug;
+
+  return (
+    <tr
+      key={key}
```

Comment:
`Key` prop is not required for a `tr` element, unless the row is part of an array being iterated over. In this case, the `key` should be passed at the time of iteration, not inside the individual component.